packages/react-component-library/src/components/Timeline/__tests__/Timeline.test.tsx

```diff
@@ -2374,6 +2374,121 @@ describe('Timeline', () => {
     })
   })
 
+  describe('when `startDate`  and `endDate` is updated externally', () => {
```

Comment:
Minor: extra space here. 
```suggestion
  describe('when `startDate` and `endDate` is updated externally', () => {
```